packages/framework/src/parser.ts

```diff
@@ -462,3 +465,39 @@ const unwrapLanguage = (language: LanguageDTO) => {
     resources: parse(language.content) as { [key: string]: unknown },
   };
 };
+
+const unwrapMenu = (
```

Comment:
What is this the purpose of this function vs `parseMenu`? We should just re-use parseMenu and return a EnsembleMenuModel.